ci-operator/jobs/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master-presubmits.yaml

```diff
@@ -121,3 +121,54 @@ presubmits:
         secret:
           secretName: regcred
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
```

Comment:
All this block must be moved in a separate job in the same folder (ie `openshift-metal3-dev-scripts-packetcheck.yaml`), otherwise it will get duplicated

Reply:
This job file was auto-generated by "make jobs". My understanding of DPTP/prow logic is any changes to be done in ci-operator config yaml, not in job directly. Otherwise config check will fail.

Reply:
This is being generated since you have an `images` stanza in the `__packetcheck.yaml` configuration.

Reply:
@stevekuznetsov That was it, thank you!